website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -10,7 +10,7 @@ Run Tasks allow you to directly integrate third-party tools and services at cert
 
 You can manage run tasks through the UI or through the [Run Tasks APIs](/cloud-docs/api-docs/run-tasks/run-tasks).
 
-> **Hands-on:** Try the [Configure Snyk Run Task in Terraform Cloud](https://learn.hashicorp.com/tutorials/terraform/cloud-run-tasks-snyk?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
+> **Hands-on:** Try the [HCP Packer validation run task](https://developer.hashicorp.com/packer/tutorials/hcp/setup-tfc-run-task) tutorial.
```

Comment:
```suggestion
> **Hands-on:** Try the [HCP Packer validation run task](/packer/tutorials/hcp/setup-tfc-run-task) tutorial.
```

Modified the link. Tutorials have their own repository now, so we don't need the full developer.hashicorp.com link for tutorials anymore.